packages/manager/modules/pci/src/projects/project/quota/increase-request/translations/Messages_fr_CA.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_quota_increase_title": "Augmenter mes quotas",
+  "pci_projects_project_quota_increase_submit_label": "Confirmer",
+  "pci_projects_project_quota_increase_cancel_label": "Annuler",
+  "pci_projects_project_quota_increase_content": "Êtes-vous sûr de vouloir augmenter les quotas de votre projet <strong>{{ projectId }}</strong> ?",
+  "pci_projects_project_quota_increase_success_message": "Votre demande d'augmentation de quota a bien été prise en compte. <a class=\"oui-link_icon\" target=\"_blank\" href=\"{{ ticketUrl }}\">Cliquez ici pour suivre son évolution. <span class=\"oui-icon oui-icon-external_link\" aria-hidden=\"true\"></span></a>",
+  "pci_projects_project_quota_increase_error_message": "Une erreur est survenue lors de la demande d'augmentation de vos quotas : {{ message }}."
+}
```

Comment:
We normally provide only `fr_FR` translations as others will be overwritten by translation script